server/app/src/androidTest/java/sh/calaba/instrumentationbackend/StatusReporterObject.java

```diff
@@ -0,0 +1,76 @@
+package sh.calaba.instrumentationbackend;
+
+import android.os.Environment;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+
+public class StatusReporterObject {
+
+    public static void report(String method,
+                              String message,
+                              StatusReporter.FinishedState extraState,
+                              boolean hasData) {
+
+
+        System.out.println("Failure file: " + getOutputFile(FAILURE_FILE_PATH));
+        System.out.println("Finished file: " + getOutputFile(FINISHED_FILE_PATH));
+
+        if (hasData) {
+
+            System.out.println("method: " + method);
+
+            if ("report-failure".equals(method)) {
```

Comment:
I believe it makes sense to move method name in symbolic constant like `REPORT_FAILURE_METHOD` and use it here and in `./StatusReporter.java` file

Reply:
Agree on that. @marsinchevskaya please pay attention to this